### PR TITLE
fix: Wrong share text on farcaster

### LINKED ISSUE #9982: Wrong share text on farcaster
image
I've minted a generative art piece on Koda #115-2404202980/256

also minted this: 
image

this number 2404202980 is redundant 

so something is broken there

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- fixed sn numbers
- added collection frame url
- [x] Closes #9982

## Needs QA check

- @kodadot/qa-guild please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI;

### CODE REVIEW

**Comment by prury** [comment]
can we ditch the total number of pieces on the share text? at least for me its a bit confusing wdyt

**Comment by hassnian** [comment]
> can we ditch the total number of pieces on the share text? at least for me its a bit confusing wdyt

updated 

CleanShot 2024-05-04 at 11 23 50@2x

File: locales/en.json [hunk 1]
```diff
@@ -1099,6 +1099,7 @@
     "listedNfts": "Check out these awesome NFTs I just listed on KodaDot:\n {0}",
     "nftWithArtist": "Check out this awesome NFT on {'@'}KodaDot from {'@'}{0}",
     "dropNft": "I've minted a generative art piece on Koda {'#'}{0}{'/'}{1}",
+    "dropNfts": "I've minted {0} generative art pieces on Koda",
```
Comment (exezbcz) [inline locales/en.json:1102]: Koda! 


will make it more exciting

**Review by exezbcz — CHANGES_REQUESTED** [review]

**Review by Jarsen136 — APPROVED** [review]

File: locales/en.json (hunk 1 above)
Comment (hassnian) [reply locales/en.json]: updated 🫡

**Comment by prury** [comment]
@hassnian i can still see the total collection amount here when minting one:

image

**Comment by hassnian** [comment]
> @hassnian i can still see the total collection amount here when minting one:
> 
> image

fixed

CleanShot 2024-05-06 at 18 42 21@2x

**Review by exezbcz — APPROVED** [review]

File: components/common/successfulModal/ShareSocialsSection.vue [hunk 2]
```diff
@@ -39,12 +39,18 @@ import { NeoButton, NeoIcon, NeoTooltip } from '@kodadot1/brick'
...
+  farcaster?: { embeds: string[] }
+}
+
 const props = withDefaults(
   defineProps<{
     text: string
     url: string
     withCopy?: boolean
     showFarcaster?: boolean
+    social?: SocialMediaProps
```
Comment (preschian) [inline components/common/successfulModal/ShareSocialsSection.vue:53]: eslint, default value

**Review by preschian — APPROVED** [review]

File: components/common/successfulModal/ShareSocialsSection.vue (hunk 2 above)
Comment (hassnian) [reply components/common/successfulModal/ShareSocialsSection.vue:53]: updated

**Pull request merged**